app/ui/legacy/src/main/java/com/fsck/k9/activity/EditIdentity.kt

```diff
@@ -87,6 +90,20 @@ class EditIdentity : K9Activity() {
         outState.putParcelable(EXTRA_IDENTITY, identity)
     }
 
+    override fun onCreateOptionsMenu(menu: Menu?): Boolean {
+        menuInflater.inflate(R.menu.edit_identity_menu, menu)
+        return true
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        if (item.itemId == android.R.id.home || item.itemId == R.id.save) {
```

Comment:
If there's a 'save' button using up or the back button should not also save the changes.